lib/src/commands/get/team_entry.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:args/command_runner.dart';
+class TeamEntryCommand extends Command {
+  @override
+  String get name => 'team-entry';
+
+  @override
+  String get description => 'Add team entry to database';
+
+  var databaseHandler;
+  
+  // @override
+  // String get usageFooter => 'This is usage';
+
+   @override
+  String get usage => ("$description\n\n" + 
+"""Usage: vscout add team-entry [team-id]
```

Comment:
Why do you need a getter for usage when it could just be generated automatically?